### PR TITLE
adding bump.sh's train travel api

### PR DESCRIPTION
**Pull request by philsturgeon**
### Description

Adding @bump-sh's Train Travel API as a demo API that folks can use.

### CODE REVIEW

**Comment by yada** [comment]
Thanks, @philsturgeon, for this very excellent contribution. 
Please sign the PR to pass the DCO test, which is mandatory ;-)

**Comment by philsturgeon** [comment]
@yada done!

File: artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  icon:
+    - base64data: iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAIAAADYYG7QAAACZUlEQVR4nOzXy2sTQRwH8NlH0hLRutYeugQiHgJBcIN4SAJpbhVqwP/AgwfBQwVrG9GKUMGTioo3e1FvPg4BKxjIyRwitehBMSJUSGzDmlAHa2v3kZ0R2uBmQ+wMtRktzPeSZPa3mU92Jvlt5P79A+B/ivivAe3hIFI4iBQOIoWDSOEgUmRihaZpx4aH2wYbTqNU+pjP523b7ngW3jOIDyZRzz7w7YtUKQjm8raBotFoJjPR8VC1Wp3IXMjlcp5RQQQjF/HRk8gCjmGjtYa5suKbvev/9IgG9FdLpqrqwwf347GYx3PiChg6DcT1j4rXh+Re48g5M5TuOggAIEmS5/oNHBBTp1p07lPj0Bks+rsOAgAkEoldgUATEB0BouQewy4L+/vs/igLkCRJexWlCdq9ye0exn6FBcg0zXq93pxzqdJJsvEgiGs6C9CLXM6yrObUb59j82cLomWm1QUZvu86CEI4NXXVfb1cQ08nAfZyBAAcK/DmmoCd7oJez80dT6crFe8yzT4GT8aB+cNdqlU9UBiXl97RvKdA/NcRCoU0TWsbRMiZn/9cKpX+dBaWe7F6GPf0Cd91of6B5trQghhnBzbXcDjc1hzo4yC0uLhQLL4yDGPbQPF4/OaN61sDbQRCODZ2/tnMDE0xiyVTFGV6+l4qlaIpZrSHZFm+c/uWz+cjVrLb1MFgMEaxF5l+y1R1kFjDFKTrX4k17EC1Wq1YLBLL2IEuTV7+fVOwSViAIISjo2ez2SxNMbmXRSKRoWRyaxSEULlcflko0P9S8+ZKCgeRwkGkcBApHEQKB5HyKwAA//8dEMgpImCtowAAAABJRU5ErkJggg==
+      mediatype: image/png
+spec:
+  version: '1.0'
+  capabilities: Full Mocks
+  contracts:
+    - type: openapi
+      url: https://raw.githubusercontent.com/bump-sh-examples/train-travel-api/main/openapi.yaml
```
Comment (lbroudoux) [inline artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml:48]: It looks like this OpenAPI file does not contain examples that can be imported directly into Microcks. Is it the correct URL? I think you were working on a different branch...

**Review by lbroudoux — CHANGES_REQUESTED** [review]
I think this is not the correct URL to final artifact that sticks to Microcks conventions. Could you please check this?

File: artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml (hunk 1 above)
Comment (lbroudoux) [reply artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml:48]: Got it! The OpenAPI lives on another branch and the URL should be https://raw.githubusercontent.com/bump-sh-examples/train-travel-api/refs/heads/feat/microcks/openapi.yaml instead. Be ware that https://docs.bump.sh/guides/bump-sh-tutorials/mocking-with-microcks/ is mentioning the OpenAPI file on `main` branch too!

Comment (philsturgeon) [reply artifacts/bump.sh/train-travel/1.0/train-travel.1.0.api.yml:48]: @lbroudoux thanks for prompting me to get on with it. I have a PR open to get the changes merged, which I've just tidied up ready for review. It's got a few questions on there which you might be interested in. https://github.com/bump-sh-examples/train-travel-api/pull/21

**Comment by philsturgeon** [comment]
@lbroudoux now that https://github.com/bump-sh-examples/train-travel-api/pull/21 is merged pointing to a good Microcks-compatible OpenAPI, and I've fixed the DCO, can you review this again?

**Comment by lbroudoux** [comment]
Yep sure!

**Review by lbroudoux — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**